lib/stack_master/aws_driver/s3.rb

```diff
@@ -48,6 +48,31 @@ def upload_files(bucket: nil, prefix: nil, region: nil, files: {})
         end
       end
 
+
+      def download_file(bucket, prefix, region, filename)
+        raise StackMaster::AwsDriver::S3ConfigurationError, 'A bucket must be specified in order to use S3' unless bucket
+
+        s3 = new_s3_client(region: region)
+        StackMaster.stdout.puts "Downloading file from S3"
+
+        downloaded_file = File.join(Dir.tmpdir(), filename)
```

Comment:
The parenthesis to `tmpdir` are optional and most style guides say you should exclude them.